frontend/src/components/NavBar.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Divider,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverHeader,
+  PopoverBody,
+  PopoverArrow,
+  PopoverCloseButton,
+  Portal,
+  Link,
+} from "@chakra-ui/react";
+import {
+  OverViewIcon,
+  ToDoIcon,
+  PendingIcon,
+  CompletedIcon,
+  HistoryIcon,
+  SettingsIcon,
+  PlusIcon,
+} from "@/static/icons";
+import { useRouter } from "next/router";
+
+// button abstraction to include the Link and styling
+const StyleButton = ({ children, link }: { children: any; link: string }) => {
+  const router = useRouter();
+  const isActive = router.pathname === link;
+  return (
+    <>
+      <Link href={link} width="100%">
+        <Button
+          justifyContent="flex-start"
+          bg="transparent"
+          height="9"
+          width="100%"
+          borderRadius={10}
+          // temporary until I figure out custom color schemes
+          style={{
+            background: isActive ? "#FAFA78" : "white",
+          }}
+          _hover={{
+            background: "#FAFA78 !important",
+          }}
+          isActive={isActive}
+        >
+          {children}
+        </Button>
+      </Link>
+    </>
+  );
+};
+
+// Navbar component
+export const NavBar: React.FC = () => {
+  return (
+    <Box padding={10} w={250}>
+      <Flex
+        alignItems="flex-start"
+        justify="space-around"
+        flexDirection="column"
+      >
+        <Popover>
+          <PopoverTrigger>
+            <Button
+              marginBottom="5"
+              height="9"
+              width="85%"
+              justifyContent="flex-start"
+              bg="#FF5000"
```

Comment:
Slightly picky but the button color here is a little different from the Figma. We use "#D74100" there

Reply:
Wait I might be dumb I was looking at the wrong version, i guess it doesn't really matter then!

Reply:
Oh one more thing cause I'm on a tangent lol should the Museum of Fine Arts, Boston be part of the nav bar or not?

Reply:
I think maybe that can be apart of the topbar with the search bar and profile icon, but not completely sure.

Reply:
I think it makes sense to make it part of the topbar